post-orientation-exercises/postgres/05-sequelize-seeding.md

```diff
@@ -1,62 +1,49 @@
-### Seeding
-
-You've seeded data before, but Sequelize is going to make the process a bit easier (although official documentation for this process is severely lacking as of this writing).
-
-Just like migrations, creating seed files with Sequleize requires the CLI tool. And, similar to migrations, you can create a seed file by running `sequelize seed:create --name my-seed-file`
-
-Sequelize will create a seed file with a timestamp and a `.js` extension. Open it up and it looks like this:
-```js
-'use strict';
-
-module.exports = {
-  up: function (queryInterface, Sequelize) {
-    /*
-      Add altering commands here.
-      Return a promise to correctly handle asynchronicity.
-
-      Example:
-      return queryInterface.bulkInsert('Person', [{
-        name: 'John Doe',
-        isBetaMember: false
-      }], {});
-    */
-  },
-
-  down: function (queryInterface, Sequelize) {
-    /*
-      Add reverting commands here.
-      Return a promise to correctly handle asynchronicity.
-
-      Example:
-      return queryInterface.bulkDelete('Person', null, {});
-    */
-  }
-};
-```
+### Seeding with BulkCreate()
 
-Update the commented-out example code with the table name you want to populate (change 'Person' to 'Orders', for example). Then pass in an array of objects that you want to insert into the db as the second argument.
-```js
-const { shows } = require('./data/shows'); //some random json file
-return queryInterface.bulkInsert('Shows', shows, {});
-```
+You've seeded data before, but Sequelize is going to make the process a bit smoother.
+
+Inside of the function that we just used to create the tables, we can now insert the necessary method to populate them with some data. This is the bulkCreate method on queryInterface.
 
-To seed your database, use `sequelize db:seed:all` to run all the seed files or `sequelize db:seed --seed seed_file_name` to seed one file. With `all`, the migrations are run by timestamps in ascending order. If you rename the files without the timestamps, you can name them in alphabetical order and they will run from a - z.
+As before, you will need a json file where the keys match exactly with the column names of your db table. Import these from wherever they live, then you can bulkCreate them right into the tables.
+
+```
+let sequelize = require('sequelize');
+let queryInterface = require('sequelize/lib/query-interface');
+
+const { users } = require('./data/users.json')
+const { Computers } = require('./data/computers.json')
+
```

Comment:
One of these destructured props is caps and the other isn't. Let's make them match